src/json.ts

```diff
@@ -16,11 +29,23 @@ export declare type JSONValue<T extends any = string | number | boolean> =
  * @param init optional configurations for `Response` and `JSON`
  * @returns
  */
-export function json<T extends any = JSONValue>(data: T, {
+export function json<T extends any>(data: T, {
+  replacer,
+  space,
+  ...init
+}?: Partial<JSONInit>): Response;
+
+export function json<T extends any>(data: T, {
+  replacer,
+  space,
+  ...init
+}?: Partial<JSONInitAlt>): Response;
+
+export function json<T extends any>(data: T, {
```

Comment:
Add an additional overload to accept the format `json(data, status)`, e.g. `json({}, 200)`